src/api/AciContractCallEncoder.js

```diff
@@ -39,7 +39,7 @@ class AciContractCallEncoder {
      * const data = encoder.decodeCall('Test', 'test_string', 'cb_KxHwzCuVGyl3aG9vbHltb2x5zwMSnw==')
      * console.log(`Decoded data: ${data}`)
      * // Outputs:
-     * // Decoded data: ["whoolymoly"]
+     * // Decoded data: { functionId: "f0cc2b95", args: ["whoolymoly"] }
```

Comment:
leftover ?

Reply:
nope, this example is not correct. I see this format returned in tests:
https://github.com/aeternity/aepp-calldata-js/blob/6cb6f2bfd989725afff69f4ae460e867fc439cf2/tests/AciContractCallEncoder.js#L61-L83